desktop/core/src/desktop/js/onePageViewModel.js

```diff
@@ -442,13 +442,28 @@ class OnePageViewModel {
       $('.dz-drag-hover').removeClass('dz-drag-hover');
     };
 
+    let dbname = '';
+    let interpreter = '';
+
     const openImporter = function (path) {
+      const table = path.substring(path.lastIndexOf(':') + 1, path.lastIndexOf(';'));
       self.loadApp('importer');
       self.getActiveAppViewModel(vm => {
         vm.createWizard.source.path(path);
+        vm.currentStep(2);
+        vm.createWizard.source.interpreter(interpreter);
+        vm.createWizard.destination.name(dbname + '.' + table.substring(0, table.length - 4));
+        window.setTimeout(() => {
+          vm.createWizard.indexFile();
+        }, 500);
       });
     };
 
+    huePubSub.subscribe('left.assist.path', assis_path => {
+      dbname = assis_path.dbname[0];
+      interpreter = assis_path.dialect;
+    });
```

Comment:
This feels hacky: can't we instead add a pubsub event to ask for the current path instead?

Reply:
Done, now it is gone. Using huepubsub.